src/app/payment/paymentMethod/BlueSnapV2PaymentMethod.spec.tsx

```diff
@@ -122,6 +122,32 @@ describe('when using BlueSnapV2 payment', () => {
             .toHaveLength(1);
     });
 
+    it('renders modal but does not append bluesnap payment page because is empty', async () => {
```

Comment:
was this change intended?

Reply:
Yes, it was intended because yesterday I pulled from master and global coverage threshold for branches was not met and circleci was complaining. But now seems to be working again.

Reply:
I opened up a new PR: https://github.com/bigcommerce/checkout-js/pull/600